.storybook/decorators/helpers.js

```diff
@@ -70,3 +70,16 @@ export function fetchContainers(id, isDocs = false) {
 
     return containers;
 }
+
+/**
+ * Object for mapping "size" arg values to their display names.
+ */
+export const sizeDisplayNames = {
```

Comment:
This isn't necessary if we use the `Sizes` utility available as an export from `@spectrum-css/preview/decorators`; another reason to use Sizes directly is so that we use consistent grid spacing and layout and typography is loaded in a unified way (instead of loading Typography into components).

Reply:
This commit was dropped.